include/rapid_pbd/motion_planning.h

```diff
@@ -38,6 +38,7 @@ class MotionPlanning {
   void ClearGoals();
   void BuildGoal(moveit_msgs::MoveGroupGoal* goal) const;
   int num_goals() const;
+  void publishCollisionObject(moveit_msgs::CollisionObject&);
```

Comment:
Name this `PublishCollisionObject` as per the [Google C++ Style Guide](http://google.github.io/styleguide/cppguide.html). It should also take the argument as a const ref, and it should have a name, like this:

```cpp
void PublishCollisionObject(const moveit_msgs::CollisionObject& obj);
```